lib/server/generate.js

```diff
@@ -399,6 +404,24 @@ async function execute() {
       }
 
       fs.writeFileSync(mainCss, cssContent);
+    } else if (normalizedFile.match(/\.png$|.jpg$|.svg$|.gif$/)) {
+      console.log(normalizedFile);
+      let parts = normalizedFile.split(sep + 'static' + sep);
+      let targetDirectory = join(
```

Comment:
Use `const`.

Reply:
Was following the same convention used on the next block, should I update that one as well?

Reply:
Sure!